doc/architecture/decisions/0005-use-a-guid-as-installation-id.md

```diff
@@ -0,0 +1,28 @@
+# 5. Use a GUID as Installation ID
+
+Date: 2018-04-27
+
+## Status
+
+Accepted
+
+## Context
+
+When installing a new device in the Notification Hub an Installation ID has to be provide by the App. Notification Hub
```

Comment:
Now that we're using "installation" in place of "device", all wording should follow... "When adding a new installation..."

Reply:
also "provide" -> "provided"